src/components/common/SessionAccounts.vue

```diff
@@ -1,37 +1,38 @@
 <template>
-  <div class="session-accounts">
-    <div class="session-accounts-top">
-      <h2>My accounts</h2>
-      <p>
-        You can use this account to explore
-        <router-link to="/wallet">
-          Lunie
-        </router-link>
-        and to approve transactions.
-      </p>
-    </div>
-    <div v-for="account in accounts" :key="account.address">
-      <div class="card content">
-        <div class="content-left">
-          <h3>{{ account.name }}</h3>
-          <Bech32 :address="account.address" short-form />
+  <SessionFrame>
+    <div class="session">
+      <div class="session-container">
+        <h2 class="session-title">My accounts</h2>
+        <div class="session-main">
+          <p>
+            You can use the accounts below to explore
+            <router-link to="/wallet">
+              Lunie
+            </router-link>
+            and to approve transactions.
+          </p>
+
+          <AccountList :accounts="accounts" />
+
+          <div class="button-add-account">
+            <TmBtn value="Add Another Account" to="/accounts" color="primary" />
```

Comment:
Do you need a `type="link"` on this button? Otherwise it won't become a `router-link`?

Reply:
yeah that's correct - but this will display as a button that links to `/accounts`

Reply:
Ok, but I believe the `TmBtn` component only handles the `to` prop when it is also a `type === `'ink'`.